docs/fit_spectrum.rst

```diff
@@ -2,8 +2,113 @@
 How to fit a spectrum
 #####################
 
-Details of how to fit a spectrum.
+An observed spectrum can be fit and plotted using command line scripts.
 
-From commandline.
+Fitting
+=======
 
-From code.
+A spectrum can be fit from the command line using the ``run_pahfit`` script.
+To run this script, the observed spectrum to be fit and a model pack file
+are specified.
+
+For example, to fit the Spitzer IRS SL/LL spectrum of the M101 nucleus
+(included in the data directory, from
+`Gordon et al. 2008, ApJ, 682, 336 <https://ui.adsabs.harvard.edu/#abs/2008ApJ...682..336G/abstract>`_),
+the command is:
+
+.. code-block:: console
+
+  $ run_pahfit M101_Nucleus_irs.ipac scipack_ExGal_SpitzerIRSSLLL.ipac
+
+The first argument gives the file with the observed spectrum.
+The second argument gives the file with the model pack file to be used.
+
+The ``run_pahfit`` script first looks in the current directory for the
+model pack file, then looks in the installed repository location for the
+file in the packs/ subdirectory.  Thus, a default pack can be overridden
+by copying it into the current directory and modifying it.
+
+By default, the ``run_pahfit`` script will save the output as an ASCII
+IPAC table.  In addition, the plot of the observed spectrum with the
+best fit model (including components) is saved as a pdf file.  The format
+of the output and plot files can be changed through command line arguments.
+
+Help for the possible command line options for the ``run_pahfit`` script
+can be seen by:
+
+.. code-block:: console
+
+  $ run_pahfit --help
+
+Example Output
+==============
+
+The best fit model as well as the constraints for each PAHFIT model component
+are saved in the output file.  For the example above, the output file in IPAC
+table format is below.  A value of 'null' means that parameter not used
+by that component and a value of 'nan' means that there is no constraint
```

Comment:
Same here, the quotes around nan output weird characters in the http://www.stsci.edu/~kgordon/pahfit/fit_spectrum.html page

Reply:
Hopefully fixed in the next commit.